services/meta/data.go

```diff
@@ -576,9 +585,12 @@ func (data *Data) DropUser(name string) error {
 	for i := range data.Users {
 		if data.Users[i].Name == name {
 			data.Users = append(data.Users[:i], data.Users[i+1:]...)
+			// Maybe we dropped the only admin user?
+			data.adminUserExists = data.hasAdminUser()
```

Comment:
I meant if the user isn't an admin, there's not a point in refreshing the cached value